resources/views/auth/passwords/email.html.twig

```diff
@@ -10,127 +10,77 @@
-
-        </div>
-
-        <form
-            action="{{ routes.password.email }}"
-            class="form__wrapper"
-            method="POST">
-
-            {{ csrf_field() }}
-
-            <div
-                class="auth-content">
-
-                <div
-                    class="auth-scroll-wrapper">
-
-                    <div
-                        class="auth-copy">
-
-                        <div
-                            class="flex-grid">
-
-                            {% if session('status') is not empty %}
-                                <div class="box full">
-                                    <span class="auth-success">
-                                        {{ session('status') }}
-                                    </span>
-                                </div>
-                            {% endif %}
-
-                            {% if errors.has('email') %}
-                                <div class="box full">
-                                    <span class="auth-alert">
-                                        <strong>{{ errors.first('email') }}</strong>
-                                    </span>
+    {% include "common/goc" %}
+    <div data-clone>
+        <section class="tc-auth" data-c-padding="tb(triple)" data-c-overlay="black(80)" style="background-image: url('/images/bg_crowd.jpg');">
+            <div class="tc-auth-wrapper" data-c-padding="tb(triple)" data-c-container="small">
+                {# Return to the homepage link. #}
+                <a data-c-color="white" href="{{ routes.home }}" title="Return Home">
+                    <i class="fas fa-chevron-left" data-c-margin="right(quarter)"></i>{{ forgot_password.return }}
+                </a>
+                <div data-c-radius="rounded" data-c-margin="top(normal)">
+                    {# Auth UI Title Area #}
+                    <div class="tc-auth-header gradient-left-right" data-c-padding="all(double)" data-c-align="base(center)">
+                        <h1 data-c-font-size="h2" data-c-color="white">{{ forgot_password.header }}</h1>
+                    </div>
+                    {# Forgot Password Form #}
+                    <form action="{{ routes.password.email }}" method="POST">
+                        {{ csrf_field() }}
+                        <div data-c-background="white(100)" data-c-padding="tb(normal) rl(double)">
+                            <div data-c-padding="top(normal)">
+                                {% if session('status') is not empty %}
+                                    <div class="box full">
+                                        <span class="auth-success">
+                                            {{ session('status') }}
+                                        </span>
+                                    </div>
+                                {% endif %}
+                                {% if errors.has('email') %}
+                                    {# Clone alert box for errors. #}
```

Comment:
Looking like this should be split out into a partial if we're reusing the same markup across templates. Either that or it should be included as a template that gets extended.

Reply:
Good point. I'll extract it into a template.

Reply:
Localized and extracted into an error component.